package.json

```diff
@@ -18,11 +18,16 @@
     "morgan": "^1.10.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-dropzone": "^14.2.3",
     "react-icons": "^4.8.0",
     "react-redux": "^8.0.5",
     "react-router-dom": "^6.9.0",
+<<<<<<< HEAD
+    "react-toastify": "^9.1.2"
+=======
     "react-toastify": "^9.1.1",
```

Comment:
merge conflicts were not resolved blocking npm install.
removed the newer version of react-toastify.